pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -666,6 +672,7 @@ func convertToPrincipals(bindings []*rbacproto.ServiceRoleBinding, forTCPFilter
 	return enforcedPrincipals, permissivePrincipals
 }
 
+// TODO: Refactor first class fields.
```

Comment:
Put a name in the TODO. Same everywhere.

Reply:
Done. Thanks!